shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -1207,11 +1261,38 @@ - (void)updateMarkedRect:(NSDictionary*)dictionary {
 - (void)showTextInput {
   _activeView.textInputDelegate = _textInputDelegate;
   [self addToInputParentViewIfNeeded:_activeView];
+  // Adds a delay to prevent the text view from receiving accessibility
+  // focus in case it is activated during semantics updates.
+  //
+  // One common case is when the app navigates to a page with an auto
+  // focused text field. The text field will activate the FlutterTextInputView
+  // with a semantics update sent to the engine. The voiceover will focus
+  // the newly attached active view while performing accessibility update.
+  // This results in accessibility focus stuck at the FlutterTextInputView.
+  if (!_enableFlutterTextInputViewAccessibilityTimer) {
+    _enableFlutterTextInputViewAccessibilityTimer =
+        [[NSTimer scheduledTimerWithTimeInterval:kUITextInputAccessibilityEnablingDelaySeconds
+                                          target:self
+                                        selector:@selector(enableActiveViewAccessibility:)
+                                        userInfo:nil
+                                         repeats:NO] retain];
+  }
   [_activeView becomeFirstResponder];
 }
 
+- (void)enableActiveViewAccessibility:(NSTimer*)time {
+  if (_activeView.isFirstResponder) {
+    _activeView.accessibilityEnabled = YES;
+  }
+  _enableFlutterTextInputViewAccessibilityTimer = nil;
```

Comment:
there needs to be a release here... maybe it would just be easier if you used a property for this.

Reply:
thanks for reminding.